changelog.txt

```diff
@@ -1 +1,2 @@
-feature - Allow specifying retry policies for event triggered functions.
+fixed - Upgrade lodash dependency to resolve security vulnerability CVE-2019-10744
```

Comment:
While you're in here do you mind adding a period to the end of that sentence? I missed that in my review of that change.

Reply:
31952e2b4ef28b7535d0230e4568d59c8822bb3c